node/sign_js_only_package/action.yml

```
@@ -0,0 +1,91 @@
+name: Compress, Sign and Upload to GH Release
+description: 'Compresses package and signs with garasign and uploads to GH release'
+
+inputs: 
+  aws_role_arn:
+    description: 'AWS role input for drivers-github-tools/gpg-sign@v2'
+    required: true
+  aws_region_name:
+    description: 'AWS region name input for drivers-github-tools/gpg-sign@v2'
+    required: true
+  aws_secret_id:
+    description: 'AWS secret id input for drivers-github-tools/gpg-sign@v2'
+    required: true
+  npm_package_name:
+    description: 'The name for the npm package this repository represents'
+    required: true
+  dry_run:
+    description: 'Should we upload files to the release?'
+    required: false
+    default: 'true'
+  artifact_directory:
+    description: The directory in which to output signatures.
+    default: artifacts
+  sign_native:
+    description: Download and sign native packages
+    default: "false"
```

```suggestion
    default: 'false'
```

just inconsistent, wanna run prettier over all the yaml?